getting-started-service-registry/README.adoc

```diff
@@ -0,0 +1,197 @@
+[id="chap-getting-started-service-registry"]
+= Getting Started with {product-long}
+ifdef::context[:parent-context: {context}]
+:context: getting-started-service-registry
+
+////
+START GENERATED ATTRIBUTES
+WARNING: This content is generated by running npm --prefix .build run generate:attributes
+////
+
+
+:community:
+:imagesdir: ./images
+:product-long: Red Hat Openshift Service Registry
```

Comment:
Suggest: 
:product-long: Red Hat OpenShift Service Registry

Reply:
Attributes are auto-generated and need to be added separately in https://github.com/redhat-developer/app-services-guides/blob/main/_artifacts/document-attributes.adoc. Will do this in separate PR...